src/index.html

```diff
@@ -15,8 +15,93 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.google.com/specimen/Roboto"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <a
+          class="logo"
+          href="index.html"
+        >
+          <img
+            src="/src/images/logo.png"
+            alt="moyo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="ul__nav">
+            <li class="li__nav">
+              <a
+                href="#section-1"
+                class="nav__link is-active"
+              >
+                APPLE
+              </a>
+            </li>
+            <li class="li__nav">
+              <a
+                href="#section-2"
+                class="nav__link"
+              >
+                SAMSUNG
+              </a>
+            </li>
+            <li class="li__nav">
+              <a
+                href="#section-3"
+                class="nav__link"
+              >
+                SMARTPHONES
+              </a>
+            </li>
+            <li class="li__nav">
+              <a
+                href="#section-4"
+                data-qa="hover"
+                class="nav__link laptops"
+              >
```

Comment:
The class `laptops` does not follow the BEM naming convention. It should be something like `nav__link--laptops` if it's a modifier.